customer/api.go

```diff
@@ -132,8 +132,8 @@ func (a *API) ResumeChat(initialChat *InitialChat, continuous, active bool) (thr
 	return resp.ThreadID, resp.EventIDs, err
 }
 
-// ListChats returns chat summaries list.
-func (a *API) ListChats(sortOrder, pageID string, limit uint) (summary []ChatSummary, total uint, previousPage, nextPage string, err error) {
+// It returns chat info for the chats a Customer participated in.
```

Comment:
```suggestion
// ListChats returns chat info for the chats a Customer participated in.
```
Let's be consistent with all the other comments (and v3.7):
https://github.com/livechat/lc-sdk-go/blob/cc0f963a1a36d425a8ee1e107b92c9852de8667a/customer/api.go#L135